dev/tasks/linux-packages/yum/arrow.spec.in

```diff
@@ -200,10 +272,80 @@ Libraries and header files for Apache Arrow Datasets.
 %files dataset-devel
 %defattr(-,root,root,-)
 %doc README.md LICENSE.txt NOTICE.txt
+%{_includedir}/arrow/dataset/
 %{_libdir}/libarrow_dataset.a
 %{_libdir}/libarrow_dataset.so
 %{_libdir}/pkgconfig/arrow-dataset.pc
 
+%if %{use_flight}
+%package flight-libs
+Summary:	C++ library for fast data transport.
+License:	Apache-2.0
+Requires:	%{name}-libs = %{version}-%{release}
+Requires:	%{name}-python-libs = %{version}-%{release}
+Requires:	openssl
+
+%description flight-libs
+This package contains the libraries for Apache Arrow Flight.
+
+%files flight-libs
+%defattr(-,root,root,-)
+%doc README.md LICENSE.txt NOTICE.txt
+%{_libdir}/libarrow_flight.so.*
+%{_libdir}/libarrow_python_flight.so.*
+
+%package flight-devel
+Summary:	Libraries and header files for Apache Arrow Flight.
+License:	Apache-2.0
+Requires:	%{name}-flight-libs = %{version}-%{release}
+Requires:	%{name}-python-devel = %{version}-%{release}
+
+%description flight-devel
+Libraries and header files for Apache Arrow Flight.
+
+%files flight-devel
+%defattr(-,root,root,-)
+%doc README.md LICENSE.txt NOTICE.txt
+%{_includedir}/arrow/flight/
+%{_libdir}/libarrow_flight.a
+%{_libdir}/libarrow_flight.so
+%{_libdir}/libarrow_python_flight.a
+%{_libdir}/libarrow_python_flight.so
+%{_libdir}/pkgconfig/arrow-flight.pc
+%endif
+
+%if %{use_gandiva}
+%package -n gandiva-libs
+Summary:	C++ library for compiling and evaluating exressions on Apache Arrow data.
+License:	Apache-2.0
+Requires:	%{name}-libs = %{version}-%{release}
+Requires:	ncurses-libs
```

Comment:
`Requires ncurses-libs`, I'm curious why it needs ncurse.

Reply:
I thought so too.
LLVM requires `-ltinfo`. `libtinfo` is provided by `ncurses-libs`.
`LLVMSupport` CMake target has `tinfo` in its `INTERFACE_LINK_LIBRARIES` property.
And `LLVMCore` CMake target has `LLVMSupport` in its `INTERFACE_LINK_LIBRARIES` property.
And Gandiva uses `LLVMCore`.